### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by minwoox**
- Brave 5.13.11 -> 5.14.0
- Dropwizard2 2.1.1 -> 2.1.2
- GraphQL Java 19.0 -> 19.2
- GraphQL Kotlin 6.2.2 -> 6.2.5
- gRPC Java 1.49.0 -> 1.49.1
- Kafka 3.2.1 -> 3.3.0
- Kotlin 1.7.10 -> 1.7.20
- Micrometer 1.9.3 -> 1.9.4
- Netty 4.1.79.Final -> 4.1.82.Final
  - io_uring 0.0.14.Final -> 0.0.15.Final
- Reactor 3.4.22 -> 3.4.23
- Sangria GraphQL 3.2.0 -> 3.3.0
- Scala 2.12.16 -> 2.12.17, 2.13.8 -> 2.13.9
- Spring Boot 2.7.3 -> 2.7.4
- Build and Shaded
  - Akka 2.6.19 -> 2.6.20
  - Dagger 2.43.2 -> 2.44
  - dgs 5.0.5 -> 5.2.4
  - Fastutil 8.5.8 -> 8.5.9
  - GAX Java 2.19.0 -> 2.19.1
  - graphql-dgs-client 5.0.5 -> 5.2.4
  - Groovy XML 3.0.5 -> 3.0.12
  - JCTools 3.3.0 -> 4.0.1
  - JMH 0.6.7 -> 0.6.8
  - JSoup 1.15.2 -> 1.15.3
  - JUnit5 5.9.0 -> 5.9.1
  - KTLint 10.1.0 -> 11.0.0
  - Mockito 4.7.0 -> 4.8.0
  - OS detector 1.6.2 -> 1.7.1
  - Spring Web 5.3.22 -> 5.3.23

### CODE REVIEW

**Comment by ikhoon** [comment]
#4456 has been merged. Would you mind resolving the conflicts?

File: gradle/scripts/lib/scala.gradle [hunk 1]
```diff
@@ -58,7 +58,7 @@ configure(scala212 + scala213 + scala3) {
 
 configure(scala212) {
     dependencies {
-        implementation('org.scala-lang:scala-library:2.12.16')
+        implementation('org.scala-lang:scala-library:2.12.17')
```
Comment (ikhoon) [inline gradle/scripts/lib/scala.gradle:61]: These changes are applied to `gradle-scripts` via https://github.com/line/gradle-scripts/pull/129

**Comment by minwoox** [comment]
> https://github.com/line/armeria/pull/4456 has been merged. Would you mind resolving the conflicts?

Resolved. 😉

**Review by ikhoon — APPROVED** [review]
Thanks, @minwoox!

File: dependencies.toml [hunk 2]
```diff
@@ -659,11 +657,11 @@ javadocs = "https://www.javadoc.io/doc/ch.qos.logback/logback-classic/1.2.11/"
 [libraries.micrometer-core]
 module = "io.micrometer:micrometer-core"
 version.ref = "micrometer"
-javadocs = "https://www.javadoc.io/doc/io.micrometer/micrometer-core/1.9.1/"
+javadocs = "https://www.javadoc.io/doc/io.micrometer/micrometer-core/1.9.4/"
```
Comment (ikhoon) [inline dependencies.toml:660]: Self note: It seems useful to template the link of javadocs. For example, `https://www.javadoc.io/doc/io.micrometer/micrometer-core/{version}/` could be populated into the `https://www.javadoc.io/doc/io.micrometer/micrometer-core/1.9.4/`

**Review by jrhee17 — APPROVED** [review]
Thanks @minwoox ! 🙇 👍 🙇

**Comment by minwoox** [comment]
Let me rebuild a couple of more times

File: core/src/test/java/com/linecorp/armeria/client/endpoint/healthcheck/HealthCheckedEndpointGroupLongPollingTest.java [hunk 3]
```diff
@@ -38,11 +38,13 @@
 import com.linecorp.armeria.common.ResponseHeaders;
 import com.linecorp.armeria.common.annotation.Nullable;
 import com.linecorp.armeria.common.logging.RequestLog;
+import com.linecorp.armeria.internal.testing.FlakyTest;
 import com.linecorp.armeria.server.ServerBuilder;
 import com.linecorp.armeria.server.healthcheck.HealthCheckService;
 import com.linecorp.armeria.server.healthcheck.SettableHealthChecker;
 import com.linecorp.armeria.testing.junit5.server.ServerExtension;
 
+@FlakyTest
```
Comment (ikhoon) [inline core/src/test/java/com/linecorp/armeria/client/endpoint/healthcheck/HealthCheckedEndpointGroupLongPollingTest.java:47]: 🙇‍♂️

**Pull request merged**